src/engraving/rendering/dev/layoutcontext.cpp

```diff
@@ -73,6 +73,17 @@ bool LayoutConfiguration::isPrintingMode() const
     return score()->printing();
 }
 
+double LayoutConfiguration::fretWidth(const StaffType* tab) const
+{
+    if (!tab || !tab->isTabStaff()) {
+        return 0.0;
+    }
+
+    muse::draw::Font f = tab->fretFont();
+    f.setPointSizeF(tab->fretFontSize());
+    return muse::draw::FontMetrics::width(f, u"0");
```

Comment:
This feels quite hacky though. Fret width is highly variable from one fret mark to another (a two-digit fret is about twice as wide) so it doesn't feel like something that should be in LayoutConfiguration, which is about styles and general constants. Ideally we should just use the individual width of the fret that we are working on. If you really need a "reference" value of some sort, than I'd call this function "referenceFretWidth" or something like that and put it probably in the fret class. Though I think it'd be better to avoid this if we can

Reply:
The process of centering notes in chords needs a reference value, though it isn't important as to what that value is.  I'll remove this and we can just use `ctx.conf().noteHeadWidth()` (which is the width of the `noteheadBlack` symbol) for both staves.

Reply:
After another look, we don't need to centre frets at all.  (See chordlayout.cpp line 1641)